deploy/gen-include-tdishr.py

```diff
@@ -0,0 +1,72 @@
+
+import os
+import csv
+import shutil
+
+from subprocess import *
+
+# Move to the root of the repository
+os.chdir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+OPCODES_FILENAME = 'tdishr/opcodes.csv'
+OUTPUT_FILENAME = 'include/tdishr.h'
+
+print("Generating '{}' from '{}'".format(OUTPUT_FILENAME, OPCODES_FILENAME))
+
+opcodes_file = open(OPCODES_FILENAME) # , newline=''
+reader = csv.DictReader(opcodes_file)
+
+opcode_names = []
+for line in reader:
+    opcode_names.append(line['name'])
+
+output_file = open(OUTPUT_FILENAME, 'wt')
```

Comment:
you should use a context manager
```python
with open(...) as f:
    f.write(...)
    ...
```
instead of
```python
f = open(...)
f.write(...)
...
f.close()
```